Introductory Tests/EquationsOfMotion.py

```diff
@@ -55,8 +78,32 @@ def compute_motion(self, t: float, x: np.ndarray, risk_zone_radius: float = None
         Returns:
             (np.ndarray): K vector used for computing state at the following time step.
         """
-        a = self.compute_acceleration(x[0:3])
+        rotated_position = self.orbit_rotation(t, x[0:3])
+        a = self.compute_acceleration(rotated_position)
+
         kx = x[3:6]  
-        kv = a 
+        kv = a
+
         return np.concatenate((kx, kv))
+    
+
+    def orbit_rotation(self, t: float, x: np.ndarray) -> np.ndarray:
+        """ Rotates position x according to the analyzed body's real rotation.
+        Args:
+            t (float): Time value when position x occurs.
+            x (np.ndarray): Position of satellite expressed in three dimensions.
+        Returns:
+            x_rotated (np.ndarray): Rotated position using quaternions.
+        """
+        
+        # Compute angular velocity of spin using known rotational period.
+        spin_velocity = (2*pi)/self.body_spin_period
+
+        # Get Quaternion object for rotation around spin axis
+        q_rot = Quaternion(axis=self.spin_axis, angle=(2*pi-spin_velocity*t))
+        
+        # Rotate satellite position using q_rot
+        x_rotated = q_rot.rotate(x)
```

Comment:
now this seems like the kind of thing that it would be worth writing some tests for to make sure it works as expected. remind me when you are in the office to show you a quick and easy way to have some tests ;)